cmd/cloud/server.go

```diff
@@ -36,6 +36,7 @@ func init() {
 	serverCmd.PersistentFlags().String("database", "sqlite://cloud.db", "The database backing the provisioning server.")
 	serverCmd.PersistentFlags().String("listen", ":8075", "The interface and port on which to listen.")
 	serverCmd.PersistentFlags().Bool("cluster-supervisor", true, "Whether this server will run a cluster supervisor or not.")
+	serverCmd.PersistentFlags().Bool("group-supervisor", false, "Whether this server will run an installation group supervisor or not.")
```

Comment:
I must've missed the end of the discussion during the meeting; I thought we settled on `true` as the default for this. What was the reasoning behind a `false` default, if you don't mind my asking?

Reply:
Haha, I remember completely the opposite. @jwilander, can you weigh in with what you remember?

Reply:
I'm pretty sure @cpanato just wanted control for what was turned on and not to have stuff just start on by default. E.g. we have 4 provisioner pods right now with 1 tagged for cluster only and 3 for installation only. If this was on by default and pushed those would all start doing group stuff automatically

Reply:
Aka I recall the default being false

Reply:
Sounds good, I remembered the discussion but couldn't remember @cpanato 's opinion so thanks for the reminder of why we chose this default.

Reply:
That's what I remember as well. Basically default `true` would make stuff just start happening if they didn't manage to update the start commands on the pods in time.

Reply:
Ohhhh gotcha, okay, thanks